src/layers/Menu/NestedMenu.tsx

```diff
@@ -179,6 +181,13 @@ export const NestedMenu = forwardRef<NestedMenuRef, NestedMenuProps>(
       [onClose]
     );
 
+    const onKeyDown = useCallback((e: KeyboardEvent) => {
+      e.stopPropagation();
+      if (e.key === 'ArrowLeft' || e.key === 'Escape') {
```

Comment:
Wouldn't escape close the menu all together? I know the ExitListener often is handling this. Just calling this out.

Reply:
It is closing, but all levels together. But we need to give user ability get back to previous level